api/app/signals/apps/signals/workflow.py

```diff
@@ -106,27 +91,19 @@
         GEMELD,
         TE_VERZENDEN,
     ],
-    AFGEHANDELD_EXTERN: [
-        AFGEHANDELD,
-        GEANNULEERD,
-    ],
```

Comment:
AFGEHANDELD_EXTERN can be reached but not left

Reply:
Added `AFGEHANDELD_EXTERN`